src/tests/MissionCard.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import MissionCard from '../components/MissionCard';
+
+describe('MissionCard Component', () => {
+  it('renders MissionCard correctly with active member button', () => {
+    const props = {
+      name: 'Mission Name',
+      description: 'Mission Description',
+      reserved: true,
+      id: 'mission_id',
+    };
+
+    render(<MissionCard props={props} />);
```

Comment:
- The test is failing here also. Please, double-check this added tests file you've added here.